### ISSUE TITLE
feat: smooth animations when switching from one though to another just like theBrain

### ISSUE THREAD

**Issue by a0v0** [zsviczian/excalibrain#130]
![TheBrain_8zqXQemHh7](https://github.com/zsviczian/excalibrain/assets/62926341/76afaafa-3757-4eb0-a740-3d4ac4920d39)

**Comment by zsviczian** [comment]
Yes, this would be nice. However, I consider this eye-candy. This feature is very low on my priority list.
